spritepacks/released/briaryoung/briaryoung-tshirt-jurassic-park/mod_assets/monika/j/briaryoung_clothes_tshirt_jurassic_park.json

```diff
@@ -0,0 +1,73 @@
+{
+    "__author": "Briar Young",
+    "ex_props": {
+        "bare-shoulders": true,
+        "home": true,
+        "date": true
```

Comment:
are these exprops correct? bare-shoulders is probably wrong, but would you wear a jurassic park T-shirt on a date?

Reply:
Hah. Oops. I stole this from another `json` and forgot to change exprops.